### PR TITLE
[runtime] Skip fast TLS on ARM with bitcode

### PR DESCRIPTION
**Pull request by alexanderkyte**

### CODE REVIEW

File: mono/mini/mini-arm.c [hunk 1]
```diff
@@ -333,7 +333,7 @@ gpointer __aeabi_read_tp (void);
 gboolean
 mono_arch_have_fast_tls (void)
 {
-#ifdef HAVE_AEABI_READ_TP
+#ifdef HAVE_AEABI_READ_TP && !defined(DISABLE_FAST_TLS)
```
Comment (evincarofautumn) [inline mono/mini/mini-arm.c]: This should be:

```
#if defined(HAVE_AEABI_READ_TP) && !defined(DISABLE_FAST_TLS)
```

Comment (alexanderkyte) [inline mono/mini/mini-arm.c]: Fixed

**Comment by vargaz** [comment]
Why is this needed ? What breaks ?

**Comment by alexanderkyte** [comment]
We don't implement the mini opcodes for tls-get and tls-get-reg on llvm/ARM/linux. When I brought it up before, I was told that watchOS will disable it so I should just disable it.

**Comment by vargaz** [comment]
Can't we disable these at runtime using !mono_llvm_only ?

**Comment by alexanderkyte** [comment]
I don't recall why that didn't work, this is from a while ago. Let me check.

**Comment by luhenry** [comment]
@alexanderkyte @vargaz do we want to get this PR in or close it?

**Comment by vargaz** [comment]
Not sure why this is needed, why does mono_arch_have_fast_tls () get called when using bitcode ?

**Pull request closed without merging**